divi/src/wallet.h

```diff
@@ -65,6 +68,19 @@ class CReserveKey;
 class CScript;
 class CWalletTx;
 class CHDChain;
+class CTxMemPool;
+
+bool MoneyRange(CAmount nValueOut);
+
+/**
+ * Check if transaction will be final in the next block to be created.
+ *
+ * Calls IsFinalTx() with current block height and appropriate block time.
+ *
+ * See consensus/consensus.h for flag definitions.
+ */
+bool CheckFinalTx(const CTransaction& tx, int flags);
```

Comment:
Definitions for these two are (at this commit inside main.cpp instead of where they should per this move)

Reply:
done